Azure/sonic-buildimage
# [docker-radv] Fix startup issues

**- Why I did it**
PR https://github.com/Azure/sonic-buildimage/pull/4599 introduced two bugs in the startup of the router advertiser container.

1. References to the `wait_for_intf.sh` script were changed to `wait_for_link.sh`, but the actual script was not renamed
2. The `ipv6_found` Jinja2 variable added to the supervisor config file goes out of scope before it is read.

**- How I did it**
1. Rename the `wait_for_intf.sh` script to `wait_for_link.sh`
2. Use the Jinja2 concept of "namespace" to fix the scope issue

**- How to verify it**
Ensure all processes in the radv container start properly.

**- Which release branch to backport (provide reason below if selected)**
- [ x ] 201911
- [ x ] 202006
